src/rsvc/srv.c

```diff
@@ -854,6 +854,8 @@ ds_rsvc_dist_start(enum ds_rsvc_class_id class, daos_iov_t *id,
 	in->sai_class = class;
 	daos_iov_copy(&in->sai_svc_id, id);
 	uuid_copy(in->sai_db_uuid, dbid);
+	if (rc != 0)
+		goto out_rpc;
```

Comment:
Did you mean to write the following? Otherwise, I'm not following...

    rc = daos_iov_copy(&in->sai_svc_id, id);
    if (rc != 0)
        goto out_rpc;